README.md

```diff
@@ -143,26 +141,23 @@ var gpio = require("gpio");
 var gpio22, gpio4, intervalTimer;
 
 // Flashing lights if LED connected to GPIO22
-gpio22 = gpio.open({
-    pin: 22,
-    ready: function() {
-        intervalTimer = setInterval(function() {
-            gpio22.set();
-            setTimeout(function() { gpio22.reset(); }, 500);
-        }, 1000);
-    }
+gpio22 = gpio.open({ pin: 22 }, function(err, self) {
+    if(err) return console.log(err);
+    intervalTimer = setInterval(function() {
+        gpio22.set();
+        setTimeout(function() { gpio22.reset(); }, 500);
+    }, 1000);
 });
 
 // Lets assume a different LED is hooked up to pin 4, the following code
 // will make that LED blink inversely with LED from pin 22
-gpio4 = gpio.open({
-    pin: 4,
-    ready: function() {
-        // bind to gpio22's change event
-        gpio22.on("change", function(val) {
-            gpio4.set(1 - val); // set gpio4 to the opposite value
-        });
-    }
+gpio4 = gpio.open({ pin: 4 }, function(err, self) {
+    if(err) return console.log(err);
+
+    // bind to gpio22's change event
+    gpio22.on("change", function(val) {
+        gpio4.set(1 - val); // set gpio4 to the opposite value
```

Comment:
maybe we can shift val to booleans and use !val

Reply:
great idea